public/app/plugins/panel/heatmap/HeatmapPanel.tsx

```diff
@@ -56,11 +56,11 @@ export const HeatmapPanel = ({
 
   const info = useMemo(() => {
     try {
-      return prepareHeatmapData(data, options, theme, getFieldLinksSupplier);
+      return prepareHeatmapData(data.series, data.annotations, options, theme, getFieldLinksSupplier);
     } catch (ex) {
       return { warning: `${ex}` };
     }
-  }, [data, options, theme, getFieldLinksSupplier]);
+  }, [data.series, data.annotations, options, theme, getFieldLinksSupplier]);
```

Comment:
I changed this so it depends on series, not the wrapper (that may change more frequently than the data)

Reply:
yes, we gotta double-check all the places!

timeseries also has `props.data` in a deps array.